themes/navy/languages/en.yml

```diff
@@ -9,6 +9,8 @@ menu:
 
 index:
   get_started: Get started
+  theme_intro: Powerful and robust theming system that capable of making any website. Writing your theme using many template engines.
+  theme_link: Check out the Hexo themes
```

Comment:
intro: Choose from a huge library of spectacular, feature-packed and customizable themes. Create your own using any compatible template engines.
link: Explore themes

Reply:
Sounds great!